tiscale_connector.py

```diff
@@ -251,18 +250,20 @@ def _make_rest_call(
             try:
                 if(TISCALE_JSON_API_KEY in config):
                     # r = request_func(url, params=params, data=data, files=files, verify=config[phantom.APP_JSON_VERIFY])
-                    r = requests.post(url, files=files, headers={
-                                      'Authorization': 'Token %s' % config[TISCALE_JSON_API_KEY]})
+                    r = requests.post(url, files=files, timeout=10, headers={
+                                      'Authorization': 'Token %s' % config[TISCALE_JSON_API_KEY],
+                                      'User-Agent': 'ReversingLabs Phantom TiScale v2.2'})
```

Comment:
can we make the user agent string a constant? we seem to reuse it a couple of times